physics/kepler_orbit.hpp

```diff
@@ -32,6 +32,10 @@ class KeplerOrbit {
               Body const& secondary,
               KeplerianElements<Frame> const& elements_at_epoch,
               Instant const& epoch);
+  KeplerOrbit(MassiveBody const& primary,
+              Body const& secondary,
+              RelativeDegreesOfFreedom<Frame> const& state_vectors,
```

Comment:
`state_vectors` is not a very nice name.  I know, that's how it's called, but that doesn't make it better.  How about `primary_to_secondary`?